test/e2e_node/services/kubelet.go

```diff
@@ -266,9 +266,9 @@ func (e *E2EServices) startKubelet() (*server, error) {
 
 	// Apply test framework feature gates by default. This could also be overridden
 	// by kubelet-flags.
-	if framework.TestContext.FeatureGates != "" {
-		cmdArgs = append(cmdArgs, "--feature-gates", framework.TestContext.FeatureGates)
-		utilflag.NewMapStringBool(&kc.FeatureGates).Set(framework.TestContext.FeatureGates)
+	if len(framework.TestContext.FeatureGates) != 0 {
```

Comment:
prefer `if len(x) > 0`
semantically equivalent in this case but I think this is a more common style

Reply:
done :)